packages/firebase_in_app_messaging/pubspec.yaml

```
@@ -1,6 +1,6 @@
 name: firebase_in_app_messaging
 description: Flutter plugin for Firebase In-App Messaging.
-version: 0.1.2
+version: 0.1.3-dev.1
```

```suggestion
version: 0.2.0-dev.1
```

Shouldn't this be a 'minor' version bump like the others?